vaadin-platform-test/src/test/java/com/vaadin/LicenseCheckTest.java

```diff
@@ -47,6 +47,7 @@
         whitelist.add("http://www.mozilla.org/MPL/2.0/index.txt");
 
         // Apache
+        whitelist.add("http://apache.org/licenses/LICENSE-2.0");
```

Comment:
Fix for:
>java.lang.AssertionError: File /opt/agent/work/9b1a81aa43ab65b4/vaadin-platform-hybrid-test/target/generated-resources/licenses.xml contains the following dependencies with licenses that have not been whitelisted:   dependency 'net.jodah:failsafe' has licenses : [Apache License, Version 2.0: http://apache.org/licenses/LICENSE-2.0]
java.lang.AssertionError: File /opt/agent/work/9b1a81aa43ab65b4/vaadin-platform-hybrid-test/target/generated-resources/licenses.xml contains the following dependencies with licenses that have not been whitelisted: 
 dependency 'net.jodah:failsafe' has licenses : [Apache License, Version 2.0: http://apache.org/licenses/LICENSE-2.0]

Reply:
look good to me